megamek/src/megamek/client/ui/Messages.java

```diff
@@ -53,4 +53,12 @@ public static String getString(String key) {
     public static String getString(String key, Object... args) {
         return MessageFormat.format(getString(key), args);
     }
+
+    /**
+     * Returns the formatted internationalized text for the given key in the resource bundle,
+     * replacing occurrences of %s in the message with the contents of args.
+     */
+    public static String getFormattedString(String key, Object... args) {
+        return String.format(getString(key), args);
+    }
```

Comment:
So far this was done using "message=SomeError {0}: {1}". The call to Messages.getString looks the same in both cases I think. Is there an advantage to the %s version?

Reply:
This does not format numbers, which is necessary for ports.... and I apparently forgot to put that in the comment.